### DISCUSSION TITLE
Dependency graph doesn't recognize package.json

### DISCUSSION THREAD

**Discussion by christianblos** [community/community#22929] [New to GitHub]
<p>Hi,</p>
<p>I’m wondering why the dependency graph in my project is not showing up. I have a package.json defined and the file is valid. Here’s the link: <a href="https://github.com/innogames/dr-json/network/dependencies" rel="nofollow noopener">https://github.com/innogames/dr-json/network/dependencies</a></p>
<p>I have another project where the dependency graph is working fine. The package.json file there looks almost the same (<a href="https://github.com/christianblos/electron-ts-boilerplate/network/dependencies" rel="nofollow noopener">https://github.com/christianblos/electron-ts-boilerplate/network/dependencies</a>).</p>
<p>Has anyone of you experienced the same problem?</p>
<p>Thanks in advance!</p>

**Comment by paaragon** [comment]
<p>Hi!,</p>
<p>I’m having the same problem in my repo <a href="https://github.com/paaragon/angular-material-dashboard" rel="nofollow noopener">https://github.com/paaragon/angular-material-dashboard</a></p>
<p>Anyone knows the reason?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hello <a class="mention" href="/christianblos">@christianblos</a> and <a class="mention" href="/paaragon">@paaragon</a>,</p>
<p>Thanks so much for being part of the GitHub Community Forum. Sorry for the troubles you’ve been experiencing.</p>
<p>If you’re still having the same problem, can you please <a href="https://github.com/contact" rel="nofollow noopener">write to private support</a>? This sounds very similar to an issue we fixed last week, and someone there should be able to help you with your individual account. </p>
<p>Cheers!</p>

**Comment by christianblos** [comment]
<p>Thank you, I will do!</p>

**Comment by christianblos** [comment]
<p>Oh, I see it’s suddenly working now :slight_smile:</p>
